pkg/defaults/defaults.go

```diff
@@ -15,7 +13,7 @@ const (
 
 	// AddressScopeMax controls the maximum address scope for addresses to be
 	// considered local ones with HOST_ID in the ipcache
-	AddressScopeMax = int(netlink.SCOPE_LINK) - 1
+	AddressScopeMax = 0xfd - 1 // int(netlink.SCOPE_LINK) - 1
```

Comment:
Could we move this definition to a `defaults_linux.go` file in the same package, so we can retain the usage of the predefined `netlink.SCOPE_LINK` constant?